shared/runtime/pyexec.c

```diff
@@ -80,6 +80,10 @@ static int parse_compile_execute(const void *source, mp_parse_input_kind_t input
     nlr_buf_t nlr;
     nlr.ret_val = NULL;
     if (nlr_push(&nlr) == 0) {
+        #ifdef MICROPY_ENABLE_VM_ABORT
```

Comment:
```suggestion
        #if MICROPY_ENABLE_VM_ABORT
```

It is always defined as 0 or 1.

Reply:
Ok, thank you.